cf/api/repository_locator.go

```diff
@@ -100,7 +101,11 @@ func NewRepositoryLocator(config coreconfig.ReadWriter, gatewaysByName map[strin
 	}
 
 	logCacheURL := configV3.ConfigFile.LogCacheEndpoint
-	logCacheClient := command.NewLogCacheClient(logCacheURL, configV3, nil)
+	logCacheClient, err := logcache.NewClient(logCacheURL, configV3, nil, v7action2.NewDefaultKubernetesConfigGetter())
+	if err != nil {
+		panic("handle this error!")
```

Comment:
Now that creating the logcache client can yield an error, we need to handle that. The existing code also panic on error a few lines up, so we copied that pattern. We could also change this function and the caller function to return errors, and then bubble that out into the main func if you prefer.